client/app/profil/demande/demande.controller.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular.module('impactApp')
-  .controller('DemandeCtrl', function($scope, $state, $filter, $modal, toastr, RequestService, currentUser, request, prestations) {
+  .controller('DemandeCtrl', function($scope, $state, $filter, $modal, toastr, RequestService, currentUser, request) {
```

Comment:
On en avait parlé au début, on essaie de désendetter l'application notamment en supprimant les références à `$scope` lorsque l'on passe dans un controller, en utilisant la référence `controllerAs` et en remplaçant par `this`